pkgs/development/python-modules/daltonlens/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, setuptools
+, setuptools-git
+, numpy
+, pillow
+}:
+
+buildPythonPackage rec {
+  version = "0.1.5";
+  pname = "daltonlens";
+  disabled = pythonOlder "3.7";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-T7fXlRdFtcVw5WURPqZhCmulUi1ZnCfCXgcLtTHeNas=";
+  };
+
+  buildInputs = [
+    setuptools
+    setuptools-git
+  ];
+
+  propagatedBuildInputs = [
+    numpy
+    pillow
+  ];
+
```

Comment:
Please execute the tests https://github.com/DaltonLens/DaltonLens-Python/tree/master/tests